ringling/projects.py

```diff
@@ -0,0 +1,73 @@
+"""
+Copyright 2023 MSOE DISE Project
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import requests
+
+def get_url():
+    """
+    Get the URL for interacting with projects
+    :return: The project URL
+    """
+    return "http://localhost:8888/v1/projects"
```

Comment:
No bueno!  Don't hard-code a URL like that.  You need to take the base URL (http://localhost:8888) as a CLI parameter.  The rest (`/v1/projects`) can be hard-coded.